worker-sys/src/request_init.rs

```diff
@@ -86,6 +86,7 @@ impl Default for RequestInit {
     }
 }
 
+#[allow(clippy::manual_non_exhaustive)]
```

Comment:
This is a temporary fix for a linter warning:
```
warning: this seems like a manual implementation of the non-exhaustive pattern
  --> worker-sys/src/request_init.rs:89:1
   |
89 | #[wasm_bindgen]
   | ^^^^^^^^^^^^^^^
   |
   = note: `#[warn(clippy::manual_non_exhaustive)]` on by default
help: remove this variant
  --> worker-sys/src/request_init.rs:89:1
   |
89 | #[wasm_bindgen]
   | ^^^^^^^^^^^^^^^
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#manual_non_exhaustive
   = note: this warning originates in the attribute macro `wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)
```

It's probably because I accidentally updated the dependencies as I don't see this warning in the main branch.